### PR TITLE
Core elements styling (Code review)

### PR DESCRIPTION
**Pull request by AmineHLub**
<h1> Summary </h1>

- Setup page layout
- Add styling to core elements
- Add minor js functionalities (hide the booklist section including the border when there are no books)
- Add styling for buttons and error message

### CODE REVIEW

**Comment by DeliceLydia** [comment]
# Status: Invalid

Hello Team,

I am sorry I can't perform this review because you did not follow `git` workflows
please open a PR from a `feature branch ` to `dev branch` not master branch and
submit another CRR.

Thanks

**Comment by AmineHLub** [comment]
@DeliceLydia Hello. I know what you are saying but we were never required to create a DEV_Branch for testing purposes or idk we always do the following :
Master => feature branch =(PR)=> Master => new other feature branch =(PR)=> Master.

_"Opening pull requests_
_Once you start working on any project, initialize its main branch. Then, add changes in a feature branch(es). You will need to_ _create a pull request to compare your feature branch with the main branch and submit the link to that pull request via your_ _Dashboard. After you submit the link to your pull request you will get a code review from one of our code reviewers._

_You should not merge your pull requests to the main branch of your project without a code review that approves your changes."_

**quoted from** "https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md"

File: style.css [hunk 1]
```diff
@@ -1,23 +1,145 @@
...
+  flex-flow: column wrap;
+  align-items: center;
+}
+
+.add_book_header {
+  font-size: 30px;
+  margin-top: 2.5rem;
+  border-top: solid rgba(97, 97, 97, 0.404) 2px;
+  padding-top: 15px;
+  display: flex;
```
Comment (Lameck1) [inline style.css:61]: - [ ] [OPTIONAL] Let's strive to keep the code DRY. This property:
```CSS
display: flex
```
has been used several times in the CSS. Please refactor it to a class and apply the class in the HTML file to elements that share the same display property.

### Example code
```CSS
.d-flex {
  display: flex;
  align-items: center;
}
```

Then apply this class to HTML element that needs this behavior:

```HTML
  <section class="d-flex">
```

**Review by Lameck1 — APPROVED** [review]
Hi, @AmineHLub & @Goldinium,

I can see you were so adventurers with the styling. Awesome job team 👍🏼.
It's good that you are relative units for font sizing, keep up with that practice 👌🏼.
This milestone is now complete, feel free to merge this branch.

Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (AmineHLub) [reply style.css:61]: thank you also that's a nice trick bro thank you very much

**Pull request merged**